File: N4153.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class N4153 {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String s = br.readLine();
		
		while(!s.equals("0 0 0"))
		{
			StringTokenizer st = new StringTokenizer(s);
			int a = Integer.parseInt(st.nextToken());
			int b = Integer.parseInt(st.nextToken());
			int c = Integer.parseInt(st.nextToken());
			if(Pita(a,b,c)) {System.out.println("right");}
			else {System.out.println("wrong");}
			s = br.readLine();
		}		
	}
	public static boolean Pita(int a, int b, int c) {
		if(a>=b && a>=c)
		{
			if(a*a==b*b+c*c)
				return true;
			else
				return false;
		}
		else if(b>=a && b>=c)
		{
			if(b*b==a*a+c*c)
				return true;
			else
				return false;
		}
		else
		{
			if(c*c==a*a+b*b)
				return true;
			else
				return false;
		}
	}
}

File: N4153.kt
import java.io.*;
import java.util.*;

private val bw = BufferedWriter(OutputStreamWriter(System.`out`))

fun main(args: Array<String>) {
    init()
    solution()
}

private fun init() {
    val br = BufferedReader(InputStreamReader(System.`in`))

    while (true) {
        val st = StringTokenizer(br.readLine())
        var a: Int = st.nextToken().toInt()
        var b: Int = st.nextToken().toInt()
        var c: Int = st.nextToken().toInt()
        if (a == 0 && b == 0 && c == 0) break
        bw.write(triangle(a, b, c))
    }
}

private fun solution() {

    bw.flush()
    bw.close()
}

fun triangle(a: Int, b: Int, c: Int): String {
    var A: Int = Math.max(Math.max(a, b), c)
    var B: Int = Math.min(Math.min(a, b), c)
    var C: Int = a + b + c - A - B

    if (A * A == B * B + C * C) return "right\n"
    return "wrong\n"
}
